### ISSUE TITLE
Some keybinds don't work in mpv launched by invidtui

### ISSUE THREAD

**Issue by paulie-g** [darkhz/invidtui#41]
Config: latest Arch with invidtui from aur/invidtui-bin
Steps to reproduce: play any video with 'V', try to exit the player with 'q' (default mpv keybind for 'quit'), nothing happens

NB: thanks for creating this project

**Comment by darkhz** [comment]
Some default keybindings are blocked in an invidtui-launched mpv instance, like the quit keybinding, since all standard keybindings are usually handled within invidtui itself. To quit MPV, currently, you would use 'Shift+Q' within invidtui.

This was done so that while MPV exits, for example, the main invidtui instance would handle it and exit too.

This could be changed, however, like maybe, modifying MPV to reflect whatever defaults are set in invidtui?

**Comment by paulie-g** [comment]
> Some default keybindings are blocked in an invidtui-launched mpv instance, like the quit keybinding, since all standard keybindings are usually handled within invidtui itself. To quit MPV, currently, you would use 'Shift+Q' within invidtui.

Yeah, I gave the code a quick look and it's clear. For the 'play audio' use case this is indeed the desired behaviour. However, it's not ergonomic for the 'play video' use case, especially if invidtui is going to crash when mpv terminates.

**Comment by darkhz** [comment]
Agreed. The 'play audio' case has been tested more than the 'play video' case, so it seems. I will look into and optimize the 'play video' case as well.
